server/models/UserProfile.js

```diff
@@ -0,0 +1,47 @@
+const mongoose = require("mongoose");
+const User = require("./User");
+
+const Schema = mongoose.Schema,
+  ObjectId = Schema.ObjectId;
+
+const userProfileSchema = new Schema({
+  firstName: {
+    type: String,
+    required: true,
+  },
+  lastName: {
+    type: String,
+    required: true,
+  },
+  userId: {
+    type: ObjectId,
+  },
+  gender: {
+    type: String,
+    enum: ["MALE", "FEMALE"],
+    retuired: true,
+  },
+  dateOfBirth: {
+    type: Date,
+  },
+  email: {
+    type: String,
+    unique: true,
+    required: true,
+  },
+  contact: {
+    type: String,
+  },
+  address: {
+    type: String,
+  },
+  description: {
+    type: String,
+  },
+  added_date: {
```

Comment:
`camelCase` is JavaScript convention. Make sure to preference that in your code.